scripts/digest_inputs

```diff
@@ -168,3 +190,5 @@ if [ "$ARCH" = "arm" ]; then
 fi
 
 add_var PLATFORM
+
+add_var MINOR
```

Comment:
Adding var to GITHUB env so we can test minor parsing is done correctly in `digest_inputs.bats` unit tests - feel free to let me know if there is a better way to unit test the `get_minor_version` function (since this exports an additional var to github). I can extract the function into a separate file, but that would mean sourcing the logging file again in the new file (since i use `die` logging), then sourcing the new file in `digest_inputs`.